### PR TITLE
[SOS-2131] reboot cloudhosts as final task

### PR DESCRIPTION
**Pull request by sedunne**
We do the exact thing for containerhosts already, for similar reasons (to boot on the newer kernel prior to provisioning). The kernel is installed via Puppet, but since Puppet is triggered by Ansible, it should be installed on the server by the time the post task runs.

### CODE REVIEW

File: playbooks/tasks/cloudhost.yml [hunk 1]
```diff
@@ -31,22 +31,20 @@
 
 - name: Ensure APF Start-on-boot
   service:
-    name="apf"
-    enabled="yes"
-    state="started"
+    name: 'apf'
```
Comment (ChrisOrlando) [inline playbooks/tasks/cloudhost.yml:34]: we do name="apf" style nearly everywhere else across our ansible stuff.

i vote stick with that style unless we wanna explicitly change over, and in that case, we change over.

also style changes and functional changes should be in separate commits ideally

Comment (sedunne) [reply playbooks/tasks/cloudhost.yml]: changed to just be the reboot section

**Review by ChrisOrlando — APPROVED** [review]
